apps/user/views.py

```diff
@@ -55,22 +55,24 @@ def get_context_data(self, **kwargs):
     """Get context."""
 
     show_charts = 2
-    exam_ids = ExamAttempt.objects.filter(user=self.request.user).values_list(
-        'exam', flat=True).distinct()[:show_charts]
+    exam_ids = ExamAttempt.objects.filter(
+        user=self.request.user,
+        mode='exam').values_list('exam', flat=True).distinct()[:show_charts]
```

Comment:
You could use mode=ExamAttempt.EXAM_MODE here.
It seem that `.distinct()` is not really needed here and below.
Also `paginate_by = 10` and `show_charts = 2` looks weird.